tests/e2e/rudder-cdn-js/ruddertyper.yml

```diff
@@ -0,0 +1,14 @@
+# RudderStack RudderTyper Configuration
+# Just run `npx rudder-typer@next` to re-generate a client with the latest versions of these events.
```

Comment:
```suggestion
# Just run `npx rudder-typer` to re-generate a client with the latest versions of these events.
```

Reply:
Without the next tag, it is downloading an older version like beta.5.

Reply:
After upcoming rudder typer release this will be fixed. Locally for testing purpose we can use @next for now